.github/workflows/integration-test.yml

```diff
@@ -0,0 +1,78 @@
+name: Integration test with mediamtx stream
+
+on: [push]
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.19
+    
+    - name: Install dependencies
+      run: |
+        sudo apt-get update
+        sudo add-apt-repository universe &&
+        sudo apt-get install -y libswscale-dev libavcodec-dev libavformat-dev libavutil-dev ffmpeg libfuse2
```

Comment:
Pretty sure we can remove the *-dev libs and just apt install ffmpeg for fake rtsp cam. (Once static build PR is in). But with the current build/runtime we will indeed need these.

Reply:
Got it will keep that back of mind